src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+- Global Multi-cluster Management: VMware Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds, regardless of where they reside.
+- Full-stack Observability: VMware Tanzu Observability by Wavefront (TO) is a high-performance streaming analytics platform that supports 3D observability (metrics, histograms, traces/spans). You can collect data from many services and sources across your entire application stack, including edge sites.
+
+This solution architecture is intended for anyone responsible for designing, implementing, managing and securing edge locations, including DevOps admins, VI admins, platform operators, and security teams.
+
+## Topology
+
+There are multiple topologies that can be leveraged for edge deployments. A centralized model, edge site are managed from a central hub (Hub and Spoke topology). A decentralized model is also available where each edge site is managed locally.
+This solution guide is focused on the centralized model where a vCenter is deployed at the main datacenter location and a datacenter with vSphere cluster at each edge location. It is assumed that the vSphere infrastructure is already deployed.
+An Edge location can be a storefront (point of sale), manufacturing facility, medical office, or other IT infrastructure located in a remote, distributed site. Although these locations are important to your organization, they usually run up to 100 tenant virtual machines. Because of the critical nature of the workloads, dependencies on a Wide Area Network (WAN) connection must be limited.
+
+### Hub and Spoke
+
+In this topology, the management plane is entirely residing in the main datacenter. The centralized management plane composed of vCenter, NSX Advanced Load Balancer controller and TKG Management cluster are all in the same central location.
+Only the TKG Workload clusters are created at the edge along with Load Balancing components (Service Engines). This design provides a minimal footprint at the edge.
+Once the Edge site is deployed, Tanzu Mission Control is used for fleet management, allowing an easy way to manage and lifecycle TKG workload clusters at the edge.
+Additional components for monitoring, backup and log management can also be present in the main datacenter.
+
+![](./img/tkg-edge/tkg-edge-hub-spoke.drawio.png)
+
+#### Key Features
+
+- Minimal footprint.
+- Limited recoverability when site is disconnected
+- Quick and easy to deploy
+
+### Hybrid
+
+In this topology, only the TKG Management cluster is located in a central location.
+Each Edge site has the infrastructure management plane (vCenter, and NSX Advanced Load Balancer controllers, DNS servers and NTP server). This design provides a better availability when the edge location gets disconnected from the main datacenter.
```

Comment:
Each edge site contains the infrastructure management plane, which includes vCenter, NSX Advanced Load Balancer Controllers, DNS servers, and NTP servers. This design provides better availability ...